spec/command/try_settings_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'tmpdir'
+require File.expand_path('../../spec_helper', __FILE__)
+
+# The CocoaPods namespace
+#
+module Pod
+  describe TrySettings do
+    it 'returns an instance with empty defaults when there are no yml settings files' do
+      Dir.mktmpdir do |dir|
+        settings = TrySettings.settings_from_folder dir
+        settings.should.be.instance_of TrySettings
+        settings.pre_install_commands.should.be.nil?
+        settings.project_path.should.be.nil?
+      end
+    end
+
+    it 'returns an instance with the right defaults when there are no yml settings files' do
+      Dir.mktmpdir do |dir|
+        yaml = <<eos
+          try_pre_install:
```

Comment:
Why not use something like this:

```yaml
try:
  install:
    pre:
      - pod install
      - git submodule init
  workspace: ORStackView.xcworkspace
```

That way all plugin related things are grouped at the root.